Src/Base/AMReX_GpuRange.H

```diff
@@ -98,7 +98,12 @@ struct range_impl
 #elif defined (__SYCL_DEVICE_ONLY__)
         return iterator(m_b, m_gid, m_grange);
 #else
-        return iterator(m_b,0,1);
+        AMREX_IF_ON_DEVICE((
```

Comment:
Similar to Max_device, there is no reason to repeat this in two places.

Reply:
Fixed.